Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/LevelRootUiHandler.h

```diff
@@ -42,8 +42,11 @@ namespace AzToolsFramework
         Prefab::PrefabFocusPublicInterface* m_prefabFocusPublicInterface = nullptr;
         Prefab::PrefabPublicInterface* m_prefabPublicInterface = nullptr;
 
-        static constexpr int s_levelRootBorderThickness = 1;
-        static const QColor s_levelRootBorderColor;
-        static const QString s_levelRootIconPath;
+        QString m_levelRootIconPath = QString(":/Level/level.svg");
+        QColor m_levelRootBorderColor = QColor("#656565");
+
+        QColor m_prefabCapsuleColor = QColor("#1E252F");
+        QColor m_prefabCapsuleDisabledColor = QColor("#35383C");
+        QColor m_prefabCapsuleEditColor = QColor("#4A90E2");
```

Comment:
Any reason to remove the const on these?

Reply:
The reason I changed it is I was trying to keep consistency between PrefabUiHandler and LevelRootUiHandler.

I was confused by two styles in two classes. We can decide one to go with.

Reply:
The reason the members are not const in `PrefabUiHandler` is that the Ui Handler for `procprefabs` inherits from the prefab one and changes the colors.